examples/Getting_Started/Getting-started-with-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<p>After this code block, consider a "Summary" or "Conclusion" heading that highlights the key classes, methods, or functions that the code demonstrated.</p><p> </p><p>If there's still more journey ahead, consider a "Where to Go from Here" or "Next Steps" heading with a few bullet items that a customer might consider.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I forgot to add that an inspiration for some way to conclude the notebook is motivated by creating some consistency with NVDLI:

<https://github.com/NVDLI/notebooks/blob/master/even-easier-cuda/An_Even_Easier_Introduction_to_CUDA.ipynb>

Reply:
<p>The next step might be to check out the end-to-end example that builds a more complex serving ensemble (not sure if we have a place to link to that yet, since it's still in progress.)</p>